extensions/interactions/LogicProof/static/js/generatedParser.js

```diff
@@ -3388,7 +3388,8 @@ var logicProofParser = (function(){
     },
 
     /* Returns the parser source code. */
-    toSource: function() { return this._source; }
+    toSource: function() { return this._source; },
+    SyntaxError: null
```

Comment:
You might need to modify the instructions in the README file of this directory accordingly, since this is a generated file and is not typically edited by hand.

Reply:
I think this comment was missed.

Reply:
The SyntaxError property is added on the result variable after it's declaration due to which typescript raises compilation error: `property SyntaxError does not exist on type result`.

I tried using the [ts-pegjs plugin](https://github.com/metadevpro/ts-pegjs) for generating parser in ts but it produces the ts code which is compatible with Angular 2. So, we can use it once we migrate to Angular2.

I have added a comment in the readme for the same.